app/views/user_mailer/account_activation.html.erb

```diff
@@ -3,4 +3,4 @@
 <%= t(".content", user: @user.name).html_safe %>
 </p>
 <%= link_to t(".activate"),
-	edit_account_activation_url @user.activation_token, email: @user.email %>
+	edit_account_activation_url(@user.activation_token, email: @user.email) %>
```

Comment:
Em sai ở đâu hả anh ? Em thấy cái file này được gửi nguyên si cho người dùng mà , đó là lý do cần dùng url chứ không phải path chứ ạ ,trích trong tut nó cũng nói vậy "We’re going to use it in an email."